spec/support/auth_helper.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module AuthHelpers
+  module Extensions
+    def sign_in(user)
+      let(:auth_helpers_auth_token) do
+        public_send(user).create_new_auth_token
+      end
+    end
+  end
+
+  module Includables
+    HTTP_HELPERS_TO_OVERRIDE =
+      %i[get post patch put delete].freeze
+    # Override helpers for Rails 5.0
```

Comment:
AFAIK you're using Rails 6. Does this still apply?

Reply:
I think so 👀  At least it helped to fix the error preventing the tests from working properly.

Reply:
This is my solution which I've partly found on the internet cause we had to solve problems with authentication in specs somehow. Generally, it still applies to Rails 6.0 👍